pom.xml

```diff
@@ -47,7 +47,7 @@
     <log4j2.version>2.13.3</log4j2.version>
     <lombok.version>1.18.4</lombok.version>
     <mockito.version>2.22.0</mockito.version>
-    <pulsar.version>2.8.0-rc-202102252222</pulsar.version>
+    <pulsar.version>2.8.0-rc-202103202224</pulsar.version>
```

Comment:
Could you explain if the pulsar upgrade is necessary?

Reply:
Oh, this is not necessary, I didn't acquire the version `2.8.0-rc-202102252222` at first, maybe network problem. I'll recover the version.